src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { Fragment } from 'react';
+import SimpleBarChart from '@carbon/charts-react/bar-chart-simple';
+import StackedBarChart from '@carbon/charts-react/bar-chart-stacked';
+import GroupedBarChart from '@carbon/charts-react/bar-chart-grouped';
+import classnames from 'classnames';
+import every from 'lodash/every';
```

Comment:
This function is in base JS https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

Reply:
true that, force of habit. changed!